caravel/assets/visualizations/big_number.css

```diff
@@ -8,18 +9,21 @@
   font-weight: none;
 }
 
-.big_number text.big {
+.big_number text.big,
+.big_number_total text.big{
   stroke: black;
   text-anchor: middle;
   fill: black;
 }
 
-.big_number g.tick line {
+.big_number g.tick line,
+.big_number_total g.tick line{
   stroke-width: 1px;
   stroke: grey;
 }
 
-.big_number .domain {
+.big_number .domain,
+.big_number_total .domain{
```

Comment:
don't worry about changing it for this diff, but going forward I'm trying to get people to adopt dash case for css eg `.big-number-total`

BEM is also an interesting approach if you feel like reading :) https://css-tricks.com/bem-101/